django_rest_passwordreset/serializers.py

```diff
@@ -10,13 +20,52 @@
 
 
 class EmailSerializer(serializers.Serializer):
-    email = serializers.EmailField()
+    email = serializers.CharField()
+
+    def validate_email(self, value):
+        phone_number = to_python(value)
+        if phone_number and phone_number.is_valid():
+            return value
+
+        try:
+            validator = EmailValidator()
+            validator(value)
+            return value
+        except ValidationError:
+            raise ValidationError(_('Enter a valid phone number or email address.'))
```

Comment:
Conflicts with #93, which implements a universal lookup field (Not limited to phone/email)